app/src/main/res/values/strings.xml

```diff
@@ -1,25 +1,21 @@
 <resources>
     <string name="app_name">MDSApplication</string>
-    <string name="start_playing">Incepe jocul</string>
-    <string name="new_game">Joc nou</string>
-    <string name="new_game_all_caps">JOC NOU</string>
-    <string name="choose_pawn">Alege culoarea pionului</string>
-    <string name="add_members">Adauga membru</string>
+    <string name="start_playing">Start Playing</string>
+    <string name="new_game">New Game</string>
+    <string name="new_game_all_caps">NEW GAME</string>
+    <string name="choose_pawn">Choose the color of your pawn</string>
+    <string name="add_members">Add members</string>
+    <string name="add_meb_of_team">Add the members of the team...</string>
```

Comment:
Inlocuirea acestor linii nu dauneaza aplicatiei dar tot propun sa folosim limba romana in aplicatie